solidity/hardhat.config.ts

```diff
@@ -246,6 +247,12 @@ const config: HardhatUserConfig = {
   typechain: {
     outDir: "typechain",
   },
+  docgen: {
+    outputDir: "generated-docs",
+    templates: "docgen-templates",
+    pages: "single", // `single`, `items` or `files`
+    exclude: ["./test"],
```

Comment:
Will this exclude the `./test` dir or the `./contracts/test` dir?

Reply:
This is _relative to the contract sources directory_, so it applies to `./contracts/test`.
See https://github.com/OpenZeppelin/solidity-docgen/blob/master/src/config.ts for the description of the config.